### PR TITLE
Remove jQuery image loading and size checking

### PR DESCRIPTION
**Pull request by OmgImAlexis**
This will be added back in once we work out how to make it not kill loading times. Ref: https://github.com/SiCKRAGETV/sickrage-issues/issues/2723

### CODE REVIEW

File: gui/slick/views/home.mako [hunk 1]
```diff
@@ -781,12 +776,13 @@ $(document).ready(function(){
             <span class="visible-print-inline">${download_stat}</span>
         </td>
 
-        <% show_size = sickbeard.helpers.get_size(curShow._location) %>
-        <td align="center" data-show-size="${show_size}">${sickbeard.helpers.pretty_filesize(show_size)}</td>
+        ## <% show_size = sickbeard.helpers.get_size(curShow._location) %>
+        ## <td align="center" data-show-size="${show_size}">${sickbeard.helpers.pretty_filesize(show_size)}</td>
+        <td align="center" data-show-size="0">Disabled!</td>
```
Comment (MGaetan89) [inline gui/slick/views/home.mako]: Just hide the column for the moment, instead of showing "Disabled!", no?

Comment (OmgImAlexis) [inline gui/slick/views/home.mako]: I wasn't sure where the hiding was and I didn't know if we'd need to rip out half of the javascript you added. If there's an easier way to do it let me know.

Comment (MGaetan89) [inline gui/slick/views/home.mako]: I guess the easiest way would be to add `class="hidden"` (maybe check that this is the correct Bootstrap class) [here](https://github.com/SiCKRAGETV/SickRage/pull/2356/files#diff-ce131b2d456b1eaed7f0dba1e738d4e6R609), [here](https://github.com/SiCKRAGETV/SickRage/pull/2356/files#diff-ce131b2d456b1eaed7f0dba1e738d4e6R625) and on your "Disabled" cell (which can now be empty).

Comment (OmgImAlexis) [inline gui/slick/views/home.mako]: Alright, still need to have the size check commented out but that should be good.

**Comment by miigotu** [comment]
Hide it?

**Comment by OmgImAlexis** [comment]
@miigotu tried to hide it and it breaks the table layout because of tablesort. For now we could just use `&nbsp` or a space?

**Comment by miigotu** [comment]
`<td />` ?

File: gui/slick/views/home.mako [hunk 2]
```diff
@@ -392,6 +387,9 @@ $(document).ready(function(){
           % endif
         });
 
+        // Hides size column for now until we can fix it
+        $('[data-show-size]').hide();
+        $('[data-column="6"]').hide();
```
Comment (OmgImAlexis) [inline gui/slick/views/home.mako]: @MGaetan89 @miigotu "fixed" it.

Comment (MGaetan89) [inline gui/slick/views/home.mako]: Might work

Comment (OmgImAlexis) [inline gui/slick/views/home.mako]: Well it hides the columns after they've loaded, just not sure if we should hot-fix it like this or actually remove it for now?

**Pull request merged**